source/MultiGridOctreeData.inl

```diff
@@ -2653,14 +2653,6 @@ inline int Octree<Degree>::GetEdgeLoops(std::vector<std::pair<long long,long lon
 	return int(loops.size());
 }
 
-template<int Degree>
-inline int Octree<Degree>::AddTriangles(CoredMeshData* mesh,std::vector<CoredPointIndex> edges[3],std::vector<Point3D<float> >* interiorPositions,const int& offSet)
-{
-	std::vector<CoredPointIndex> e;
-	for(int i=0;i<3;i++){for(size_t j=0;j<edges[i].size();j++){e.push_back(edges[i][j]);}}
-	return AddTriangles(mesh,e,interiorPositions,offSet);
```

Comment:
This call is incorrect - it lacks the 5th argument. With GCC up to version 6 this compiles as the function is never used. With GCC 7 this leads to compilation error. An alternative to removing the method could be adding a 5th parameter to be directly passed on here.